<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1321</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tasks</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>20</y>
        <width>1253</width>
        <height>503</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Task</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="taskBox">
              <property name="minimumSize">
               <size>
                <width>280</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Select Category ...</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>All</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="taskListWidget"/>
          </item>
          <item>
           <widget class="QTextBrowser" name="taskTextBrowser"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Run job every with Cron</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Hour</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="minBox">
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="hourBox">
              <property name="maximum">
               <number>23</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="cronSetButton">
              <property name="text">
               <string>Set Cron</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="minCheckBox">
              <property name="text">
               <string>*</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="hourCheckBox">
              <property name="text">
               <string>*</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Weekday</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>MM</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="domLabel">
              <property name="text">
               <string>DD</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="dayBox">
              <item>
               <property name="text">
                <string>Select ...</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mon ...</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tue</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Wed</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Thu</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fri</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sun</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="monthBox">
              <property name="maximum">
               <number>12</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="domBox">
              <property name="maximum">
               <number>31</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Run Job Once with @</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="atSetButton">
              <property name="text">
               <string>Set @</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Run@Once</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDateTimeEdit" name="dateTimeEdit">
              <property name="minimumSize">
               <size>
                <width>268</width>
                <height>0</height>
               </size>
              </property>
              <property name="date">
               <date>
                <year>2019</year>
                <month>9</month>
                <day>9</day>
               </date>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="minimumSize">
                 <size>
                  <width>133</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Run@Now+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="atSpinBox"/>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="atComboBox">
              <item>
               <property name="text">
                <string>Select ...</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Min</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hour</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Day</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Week</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Month</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextBrowser" name="taskHelpTextBrowser"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="submitButton">
              <property name="text">
               <string>Submit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exitButton">
              <property name="text">
               <string>Exit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Past</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>1251</width>
        <height>551</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="pastListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pastButton">
           <property name="text">
            <string>View Log</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Present</string>
     </attribute>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1281</width>
        <height>551</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QListWidget" name="presentListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editButton">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionConfiguration"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>taskListSlot()</slot>
  <slot>showtaskSlot()</slot>
  <slot>helpSlot()</slot>
  <slot>submitSlot()</slot>
  <slot>deleteSlot()</slot>
  <slot>addSlot()</slot>
  <slot>editSlot()</slot>
  <slot>atShowSlot()</slot>
  <slot>cronShowSlot()</slot>
  <slot>atSetlot()</slot>
  <slot>setCronSlot()</slot>
  <slot>taskListHelp()</slot>
 </slots>
</ui>
